website/docs/zh/config/devtool.mdx

```diff
@@ -4,54 +4,85 @@ import WebpackLicense from '@components/WebpackLicense';
 
 # Devtool
 
-调试：该选项用于控制 Source Map 的生成行为。
+该选项用于控制调试信息生成策略，影响调试能力与构建性能。
+
+需要精细控制时，直接使用 [SourceMapDevToolPlugin](/plugins/webpack/source-map-dev-tool-plugin) 或 [EvalSourceMapDevToolPlugin](/plugins/webpack/eval-source-map-dev-tool-plugin)。
 
 - **类型：**
 
 ```ts
-type Devtool =
-  | false
-  | 'eval'
-  | 'cheap-source-map'
-  | 'cheap-module-source-map'
-  | 'source-map'
-  | 'inline-cheap-source-map'
-  | 'inline-cheap-module-source-map'
-  | 'inline-source-map'
-  | 'inline-nosources-cheap-source-map'
-  | 'inline-nosources-cheap-module-source-map'
-  | 'inline-nosources-source-map'
-  | 'nosources-cheap-source-map'
-  | 'nosources-cheap-module-source-map'
-  | 'nosources-source-map'
-  | 'hidden-nosources-cheap-source-map'
-  | 'hidden-nosources-cheap-module-source-map'
-  | 'hidden-nosources-source-map'
-  | 'hidden-cheap-source-map'
-  | 'hidden-cheap-module-source-map'
-  | 'hidden-source-map'
-  | 'eval-cheap-source-map'
-  | 'eval-cheap-module-source-map'
-  | 'eval-source-map'
-  | 'eval-nosources-cheap-source-map'
-  | 'eval-nosources-cheap-module-source-map'
-  | 'eval-nosources-source-map';
+type Devtool = 'string' | false;
 ```
 
 - **默认值：** `eval`
 
-Source Map 生成主要有 `source-map`、`eval`、`cheap`、`module`、`inline`、`nosources` 和 `hidden` 这几种风格，它们之间可以进行组合。
+## 决策指南
+
+### 步骤1：是否需要调试
+
+- **不需要** → 设置 `devtool: false`
+  - 禁用所有调试信息
+  - 零额外构建开销，极致构建速度
```

Comment:
This wording is more like that in a promotional article than in an official configuration document.